### ISSUE TITLE
Make the CSS framework consumable

### ISSUE THREAD

**Issue by craigcook** [mozilla/web-design-system#4]

**Comment by alexgibson** [comment]
Looking at Spark, their preferred method seems to be publishing to npm, which you can then `@import` in your CSS:

https://github.com/sparkdesignsystem/spark-design-system

If we wanted to go this route, we could probably set up an `npm publish` script, that first builds from source into a folder (such as `.dist/`), and then publishes the contents of that folder to npm.

**Comment by craigcook** [comment]
Yeah I was looking at that and it seems like a decent way to go about it. Npm could also be a way of handling versioning with tagged releases.

**Comment by michaelmccombie** [comment]
I unassigned myself from this issue and re-assigned the issue to @alexgibson . We've gotten to the point where the Protocol stylesheet is a minified CSS file. Now we need to publish the file via NPM and pull the file in to Protocol.

**Comment by alexgibson** [comment]
OK, so looking in the `./dist/` folder, what do we wish to publish to npm?

If we publish just `./dist/assets/`, then consumers would get two top level folders, `./css/` and `./js/`. Any objections, will this be enough? Would there be any other reason why sites may any of the other scaffolding?

We may also want to move `documentation.js` out of this folder and into `/demos/`, since it's not really related to Protocol itself?

**Comment by alexgibson** [comment]
Ok, I think the simplest path may be to:

- Copy Protocol assets to `/.dist/assets/protocol/`.
- Copy documentation assets to `/.dist/assets/protocol/`

We can then include a `package.json` inside `/.dist/assets/protocol/` and then publish that folder to NPM.

**Comment by craigcook** [comment]
That seems simple enough. There's also a demos folder that can be moved to .dist/assets/demos
